build/MUX-CI.yml

```diff
@@ -43,7 +43,8 @@ jobs:
     dependsOn: Build
     # To publish the package to vsts feed, set queue time variable NuGetFeed = d62f8eac-f05c-4c25-bccb-21f98b95c95f
     # This is the magic GUID from the pipeline visual designer for this feed: https://dev.azure.com/ms/microsoft-ui-xaml/_packaging?_a=feed&feed=MUX-CI
-    publishVstsFeed: $(NuGetFeed)
+    ${{ if ne(variables['NuGetFeed'], '') }}:
```

Comment:
Looks like this latest commit didn't merge master well. Can you check out the pre-merge commit and try again?

Reply:
My bad, this was from the merge commit.